src/pnpq/devices/polarization_controller_thorlabs_mpc320.py

```diff
@@ -139,6 +142,30 @@ def identify(self, chan_ident: ChanIdent) -> None:
             )
         )
 
+    def jog(self, chan_ident: ChanIdent, jog_direction: JogDirection) -> None:
+        """
+        Jogs the device forward or backwards in small steps.
+
+        The specific amount of steps per jog can be set via the PolarizationcontrollerThorlabsMPC320.set_params() function.
```

Comment:
I'm going to add Sphinx documentation generation in a PR soon. This would be a good place to try out the ability to actually make a hyperlink to the `set_params` function. Stay tuned.

Reply:
Ooh. Can't wait for the Sphinx!